pudb/settings.py

```diff
@@ -213,6 +217,13 @@ def _update_config(check_box, new_state, option_newvalue):
 
                 conf_dict.update(stringifier=newvalue)
                 _update_stringifier()
+
+        elif option == "variables_access_level":
+            # only activate if the new state of the radio button is 'on'
+            if new_state:
+                conf_dict.update(variables_access_level=newvalue)
```

Comment:
I think I'd prefer if these could be renamed `default_variables_access_level`.

Reply:
Done. I renamed all occurrences to keep it consistent.